src/lib/descriptive.ml

```diff
@@ -41,26 +53,35 @@ let skew arr =
   let std = sqrt (var arr) in
   (moment 3 arr) /. (std ** 3.0)
 
-let kurtosis arr =
-  (moment 4 arr) /. ((var arr) ** 2.0) -. 3.0
-
 let unbiased_skew arr =
   let n = float (Array.length arr) in
   (skew arr) *. (sqrt (n *. (n -. 1.0))) /. (n -. 2.0)
 
+let kurtosis arr =
+  (moment 4 arr) /. ((var arr) ** 2.0) -. 3.0
+
 let unbiased_kurtosis arr =
     let n = float (Array.length arr) in
     let k = (kurtosis arr) in
     ((n -. 1.0) *. ((n +. 1.0) *. k +. 6.0)) /. ((n -. 2.0) *. (n -. 3.0))
 
+(* See "Standard errors: A review and evaluation of standard error estimators
+  using Monte Carlo simulations" by Harding 2014, for a source of the standard
+  error calcuations. *)
 let var_standard_error arr =
-  float (Array.length arr)
+  let n = float (Array.length arr) in
+  let v = unbiased_var arr in
+  sqrt (2.0 /. (n -. 1.0)) *. v
 
 let skew_standard_error arr =
-  sqrt ( 6.0 /. (float (Array.length arr)))
+  (* Older: sqrt ( 6.0 /. (float (Array.length arr))) *)
+  let n = float (Array.length arr) in
+  sqrt ((6.0*.n*.(n-.1.0)) /. ((n-.2.0)*.(n+.1.0)*.(n+.3.0)))
```

Comment:
Styling: missing spaces.

(A whole overloading module, ``Float.(6. * n * (n - 1.) .... )`` would be nice also for readability).